resources/public/api/conf/3.0/common/pathParameters.yaml

```diff
@@ -39,3 +39,15 @@ components:
         type: string
         example: 2021-22
       required: true
+    chargeReference:
+      name: chargeReference
+      in: path
```

Comment:
This is a query parameter. And it should probably be in the queryParameters.yaml file. 
(Likewise for v2.0)